### PR TITLE
fix(tabs): Fix tabs.item icon accent color

### PR DESCRIPTION
**Pull request by EmmaOKeeffe**
## Summary

Fixes: #2319

## Release Category
Components

---

## Checklist

- [ ] MDX documentation adheres to Canvas Kit's [Documentation Guidelines](https://workday.github.io/canvas-kit/?path=/docs/guides-documentation-guidelines--page)
- [ ] Label `ready for review` has been added to PR

## For the Reviewer

- [ ] PR title is short and descriptive
- [ ] PR summary describes the change (Fixes/Resolves linked correctly)
- [ ] PR Release Notes describes additional information useful to call out in a release message or removed if not applicable
- [ ] Breaking Changes provides useful information to upgrade to this code or removed if not applicable

## Areas for Feedback? (optional)

- [ ] Code
- [ ] Documentation
- [ ] Testing
- [ ] Codemods

## Screenshots or GIFs (if applicable)
Screenshot 2023-08-22 at 11 28 29

### CODE REVIEW

**Review by RayRedGoose — APPROVED** [review]
Thank you! 🎉

File: modules/react/tabs/lib/TabsItem.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ export const StyledTabItem = styled(Box.as('button'))<StyledType & Pick<TabsItem
 
     ...hideMouseFocus,
 
-    '.wd-icon-background ~ .wd-icon-accent, .wd-icon-background ~ .wd-icon-accent2': {
+    '.wd-icon-background .wd-icon-accent, .wd-icon-background .wd-icon-accent2': {
       fill: iconColors.active,
     },
```
Comment (RayRedGoose) [inline modules/react/tabs/lib/TabsItem.tsx:105]:
```suggestion
```

File: modules/react/tabs/lib/TabsItem.tsx [hunk 2]
```diff
@@ -110,7 +110,7 @@ export const StyledTabItem = styled(Box.as('button'))<StyledType & Pick<TabsItem
       '.wd-icon-fill, .wd-icon-accent, .wd-icon-accent2': {
         fill: iconColors.hover,
       },
-      '.wd-icon-background ~ .wd-icon-accent, .wd-icon-background ~ .wd-icon-accent2': {
+      '.wd-icon-background .wd-icon-accent, .wd-icon-background .wd-icon-accent2': {
         fill: iconColors.active,
       },
```
Comment (RayRedGoose) [inline modules/react/tabs/lib/TabsItem.tsx:115]:
```suggestion
```

**Review by RayRedGoose — COMMENTED** [review]
It seems like we don't really need this styles.

**Review by fizz3r — APPROVED** [review]

**Review by RayRedGoose — DISMISSED** [review]

**Comment by RayRedGoose** [comment]
Closed because of a branch sync issue. Updated version is https://github.com/Workday/canvas-kit/pull/2323

**Pull request closed without merging**